ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -0,0 +1,704 @@
+      },
+    },
+    {
+      id: 'P4S Westerly Shift Knockback',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A00', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6A00', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6A00', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6A00', source: 'ヘスペロス', capture: false }),
+      delaySeconds: 2,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'West Knockback',
+          de: 'Rückstoß vom Westen',
+          fr: 'Poussée à l\'ouest',
+          cn: '左/西 击退',
+          ko: '서쪽 넉백',
+        },
+      },
+    },
+    {
+      id: 'P4S Acting Role',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['B6D', 'B6E', 'B6F'], capture: true }),
+      condition: Conditions.targetIsYou(),
+      infoText: (data, matches, output) => {
+        const actingRoles: { [effectId: string]: string } = {
+          'B6D': output.dps!(),
+          'B6E': output.healer!(),
+          'B6F': output.tank!(),
+        };
+        return output.text!({ actingRole: data.actingRole = actingRoles[matches.effectId] });
+      },
+      outputStrings: {
+        text: {
+          en: 'Acting ${actingRole}',
+        },
+        dps: {
+          en: 'DPS',
+        },
+        healer: {
+          en: 'Healer',
+        },
+        tank: {
+          en: 'Tank',
+        },
+      },
+    },
+    {
+      id: 'P4S Belone Bursts Position',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69D9', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '69D9', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '69D9', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '69D9', source: 'ヘスペロス', capture: false }),
+      infoText: (_data, _matches, output) => output.rolePositions!(),
+      outputStrings: {
+        rolePositions: {
+          en: 'Role positions',
+          de: 'Rollenposition',
+          fr: 'Positions par rôle',
+          ja: 'ロール特定位置へ',
+          cn: '去指定位置',
+          ko: '직업별 산개위치로',
+        },
+      },
+    },
+    {
+      id: 'P4S Belone Bursts Orbs',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69D9', source: 'Hesperos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '69D9', source: 'Hesperos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '69D9', source: 'Hespéros' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '69D9', source: 'ヘスペロス' }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 1,
+      infoText: (data, _matches, output) => output.text!({ role: data.actingRole }),
+      outputStrings: {
+        text: {
+          en: 'Share two ${role} orbs',
```

Comment:
This probably doesn't need a callout.  I think this is a "you need to know this mechanic to do it" sort of thing.

Reply:
I'll shorten it to just orbs. The role positions before is probably plenty and there is a callout for their acting role relatively near.

Reply:
Or should the role positions callout be updated to, orb role positions?

Reply:
Yeah, Belone Bursts starts casting can be "Orb Role Positions" and I think this trigger can go away.